dvc/repo/add.py

```diff
@@ -148,25 +144,41 @@ def add(  # noqa: C901
 
 
 def _process_stages(
-    repo, sub_targets, stages, no_commit, pbar, to_remote, **kwargs
+    repo, sub_targets, stages, no_commit, pbar, to_remote, to_cache, **kwargs
 ):
     link_failures = []
     from dvc.dvcfile import Dvcfile
 
     from ..output.base import OutputDoesNotExistError
 
-    if to_remote:
+    if to_remote or to_cache:
         # Already verified in the add()
         assert len(stages) == 1
         assert len(sub_targets) == 1
+        [stage], [target] = stages, sub_targets
+
+        assert len(stage.outs) == 1
+        [out] = stage.outs
```

Comment:
Could you please unpack it as a tuple for consistency? IIRC, this is what black does.

Reply:
I prefer to do it depending on the type (like use list unpacking for lists etc.) and with black the comma gets too confusing because of redundant parentheses (`(out,) = stage.outs`) though if it is the style, can comply with it.

Reply:
@isidentical Saugat is right, let's keep it consistent with the rest.